src/pages/new/volunteer/index.js

```diff
@@ -5,6 +5,10 @@ import ImageHeader from "../../../components/grid-aware/ImageHeader";
 import classroom from "../../../components/grid-aware/ImageHeader/stories/classroom.png";
 import whiteboard from "../../../components/grid-aware/ImageHeader/stories/whiteboard-sticky-notes.png";
 import StatsBlock from "../../../components/grid-aware/StatsBlock";
+import ThreeParagraphBlock from "../../../components/grid-aware/ThreeParagraphBlock";
+import teamActivityImage from "../../../components/grid-aware/ThreeParagraphBlock/stories/image4.png";
+import pairProgrammingImage from "../../../components/grid-aware/ThreeParagraphBlock/stories/image5.png";
+import teamSnackingImage from "../../../components/grid-aware/ThreeParagraphBlock/stories/image6.png";
```

Comment:
Actually, since these images are specific to this instance of the component on this page, how about we move them here to this directory?

Reply:
That would be better! Should I create an `image_assets` directory within the `src/pages/new/volunteer` directory?

Reply:
Since I don't anticipate having too many files in there, why don't we just leave it directly in the `volunteer/` directory? There's no need to add extra hierarchy unless we actually need it.